analysis/topEFT/topeft.py

```diff
@@ -261,61 +302,12 @@ def process(self, events):
         else: syst_var_list = ['nominal']
 
         # Loop over the list of systematic variations we've constructed
-        mu["pt_raw"]=mu.pt
```

Comment:
Was the raw pT simply not used anywhere?

Reply:
My understanding was that the line in question was used as a placeholder to store the "original" muon pt values so that they could be used to reset `mu["pt"]` on each pass through the `for` loop.

Since we're no longer have to re-compute the corrected muon pt for the up/down fluctuations on the Rochester corrections, I don't think we need to cache this variable anymore.

Reply:
I agree, I think that copy would no longer be needed.